internal/credentials/credentials.go

```diff
@@ -159,6 +186,10 @@ func KubeadminPasswordSecretName(clusterDeploymentName string) string {
 	return clusterDeploymentName + "-admin-password"
 }
 
+func SeedReconfigurationSecretName(clusterDeploymentName string) string {
```

Comment:
I don't think this needs to be public.

Reply:
It is used in the controller test so for not copy pasting purposing, same for other functions that create secret names